about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
optional - I would suggest that you reduce the font sizes of the text and the image to improve the UX of the webpage